### PR TITLE
Project Impact section

### PR DESCRIPTION
**Pull request by contactsanthosh21**
- Added Project impact section
- Added Color Title component
- Added Heading component
- Added Imagecard component
- Added ViewLink component

### CODE REVIEW

File: src/assets/test.png
Comment (ankitg32) [inline src/assets/test.png:1]: we can remove this i think

File: src/sections/ProjectsImpact/ProjectsImpact.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                        ControlMap was built for both beginners and experienced compliance teams. Beginners can prepare for their first audit by starting with a preset workflow that guides them through the process, complete with templates for controls, policies, and procedures
+
+                        </p>
+                        <ViewLink />
+                    </div> 
+            </div>
+         </div>
+         <div className='projects-impact__button'>
+            <Button 
+                className='projects-impact__button--child'
```
Comment (ankitg32) [inline src/sections/ProjectsImpact/ProjectsImpact.jsx]: seems like some unnecessary classes and wrappers. for eg, the div is not required when it has only one child

File: src/sections/ProjectsImpact/ProjectsImpact.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import Button from '../../components/Button/Button';
+
+var zomoLogo = require('../../assets/impact-project-1.png');
+var controlLogo = require('../../assets/impact-project-2.png');
+
+const ProjectsImpact = () => {
+  return (
+    <div className='projects-impact'>
+        <Heading className='projects-impact__heading'>The impact we have created for our clients</Heading>
+        <div className='projects-impact__wrapper'>
```
Comment (ankitg32) [inline src/sections/ProjectsImpact/ProjectsImpact.jsx]: same comment about unnecessary classes and wrappers
eg: projects-impact, projects-impact__wrapper and projects-impact__wrapper--container

File: src/sections/ProjectsImpact/ProjectsImpact.scss [hunk 3]
```diff
@@ -0,0 +1,76 @@
+@import "../../variables.scss";
+
+.projects-impact{
+    .projects-impact__heading{
+        text-align: center;
+        margin-top: 14rem;
```
Comment (ankitg32) [inline src/sections/ProjectsImpact/ProjectsImpact.scss]: should we put this at the section level instead?

**Review by ankitg32 — COMMENTED** [review]
first review

File: src/sections/ProjectsImpact/ProjectsImpact.jsx (hunk 1 above)
Comment (contactsanthosh21) [reply src/sections/ProjectsImpact/ProjectsImpact.jsx]: Done

File: src/assets/test.png
Comment (contactsanthosh21) [reply src/assets/test.png:1]: Done

File: src/sections/ProjectsImpact/ProjectsImpact.jsx (hunk 2 above)
Comment (contactsanthosh21) [reply src/sections/ProjectsImpact/ProjectsImpact.jsx]: Done

File: src/sections/ProjectsImpact/ProjectsImpact.scss (hunk 3 above)
Comment (contactsanthosh21) [reply src/sections/ProjectsImpact/ProjectsImpact.scss]: Done

File: src/components/ViewLink/ViewLink.jsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import ArrowIcon from '../../assets/ArrowIcon'
+
+const ViewLink = ({ style, link}) => {
+  return (
+    <p
+    style={{
```
Comment (ankitg32) [inline src/components/ViewLink/ViewLink.jsx:7]: similar comment as in few other places. better to give it a default style and accept className as props instead. the caller can always add styles the className being passed in props.

File: src/sections/ProjectsImpact/ProjectsImpact.jsx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+                        <ImageCard className='image' src={controlLogo} alt= "Zomo.Fit" />
+                    </div>
+                    <div className='column'>
+                        <ColorTitle className='colorTitle' backgroundColor='#F9EAF7'>Control Map</ColorTitle>
+                        <p className='description'>
+                        ControlMap is a cybersecurity compliance automation platform on a mission to simplify and reduce the stress associated with the everyday operations of modern cybersecurity compliance operations.
+                        ControlMap was built for both beginners and experienced compliance teams. Beginners can prepare for their first audit by starting with a preset workflow that guides them through the process, complete with templates for controls, policies, and procedures
+
+                        </p>
+                        <ViewLink style={{position:'absolute',bottom:0}}/>
```
Comment (ankitg32) [inline src/sections/ProjectsImpact/ProjectsImpact.jsx:42]: can we pass className and add styles to that class instead?

File: src/sections/ProjectsImpact/ProjectsImpact.scss [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+            padding: 3rem 2rem;
+            border-radius: 1.2rem;
+        }
+        .column{
+            position: relative;
+            p.description{
+                width: 36rem;
+                font-size: 1rem;
+                font-style: normal;
+                font-weight: 400;
```
Comment (ankitg32) [inline src/sections/ProjectsImpact/ProjectsImpact.scss:22]: use from variables

**Review by ankitg32 — APPROVED** [review]
minor changes, u can address them or merge as is.

File: src/sections/ProjectsImpact/ProjectsImpact.scss (hunk 6 above)
Comment (contactsanthosh21) [reply src/sections/ProjectsImpact/ProjectsImpact.scss:22]: Will take it in follow up

File: src/sections/ProjectsImpact/ProjectsImpact.jsx (hunk 5 above)
Comment (contactsanthosh21) [reply src/sections/ProjectsImpact/ProjectsImpact.jsx:42]: Will take it in follow up

**Pull request merged**